index.js

```diff
@@ -233,7 +237,29 @@ module.exports = function (options) {
 
   router.get('/', function (req, res) {
     // If we've gotten this far then token gives proper access and that's all that this route care about
-    return res.sendStatus(200);
+    var isSyndicationRequest = (req.query.q === 'syndicate-to');
+    logger.debug({'query': req.query}, 'Received a query');
+
+    if (isSyndicationRequest) {
+      res.type('application/x-www-form-urlencoded');
+      var data = '';
+      for (var i = options.syndication.length - 1; i >= 0; i--) {
+        var syndicateTo = options.syndication[i];
+        data += 'syndicate-to[]=' + syndicateTo + '&';
```

Comment:
There should be no need to do this manual appending in Node.js I believe – one can instead use the [QueryString.stringify()](https://iojs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options) method and provide it with an object.

Considering how that works I have to change the suggested response from the earlier comment to instead have `syndicate-to[]` as the key as Node.js doesn't add any `[]` when encoding a query string:

```javascript
{
  'syndicate-to[]': ['http://twitter.com/voxpelli']
}
```

If one wouldn't want to use the `QueryString.stringify()` then you would have needed to add an `encodeURIComponent()` around the `syndicateTo` as otherwise a URL that itself contains an `&` would break.